packages/react/src/themes/teams/components/Chat/chatItemStyles.ts

```diff
@@ -11,22 +13,17 @@ const chatItemStyles: ComponentSlotStylesPrepared<ChatItemProps, ChatItemVariabl
       paddingTop: pxToRem(2),
     }),
     paddingBottom: 0,
-  }),
 
-  gutter: ({ props: p, variables: v }): ICSSInJSStyle => ({
-    position: 'absolute',
-    marginTop: v.gutterMargin,
-    [p.contentPosition === 'end' ? 'right' : 'left']: 0,
-    ...((p.attached === 'bottom' || p.attached === true) && {
-      display: 'none',
-    }),
-  }),
+    [`& .${ChatGutter.className}`]: {
+      [p.contentPosition === 'end' ? 'right' : 'left']: 0,
```

Comment:
Pass props through to the ChatGutter instead of using nested selectors.  It will improve debugging, has better CSS specificity, and is inline with our current architecture statement, "We handle everything for you with shorthand, you handle everything with children."